flask_restful/__init__.py

```diff
@@ -339,8 +345,25 @@ def handle_error(self, e):
 
         if code == 401:
             resp = self.unauthorized(resp)
+
         return resp
 
+    def errorhandler(self, exception_type):
```

Comment:
We will want to allow specifying an HTTP status code as well. ([see Flask implementation](https://github.com/mitsuhiko/flask/blob/818c42cfeac4c60ba9aea2d45954ba0db9d7d06a/flask/app.py#L1135))